mlrun/runtimes/base.py

```diff
@@ -976,6 +992,46 @@ def _update_run_state(
 
         return resp
 
+    def _save_or_fire_notifications(self, runobj: RunObject):
```

Comment:
maybe we should be aligned with the terminology for sending notifications. currently from looking at the code we have 3:
- fire notification
- push
- send notification

Reply:
consolidated all to push